pkg/controller/worker/actuator.go

```diff
@@ -116,6 +128,11 @@ func NewWorkerDelegate(
 		}
 	}
 
+	dvManager, err := kubevirt.NewDefaultDataVolumeManager(providerClient)
```

Comment:
Similarly to `Actuator`, could you please inject the `DataVolumeManager` via the `WorkerDelegate` constructor, instead of creating it here? You could easily achieve that if you add it as a field to the `delegateFactory`. Then it could even be the same `DataVolumeManager` instance you inject in the `Actuator`.

Reply:
Yeah, that makes total sense, will adjust it accordingly.